server/neptune/gateway/middleware/request_id.go

```diff
@@ -0,0 +1,39 @@
+package middleware
+
+import (
+	"context"
+	"net/http"
+
+	key "github.com/runatlantis/atlantis/server/neptune/context"
+)
+
+const ghRequestIDHeader = "X-Github-Delivery"
+
+// RequestID is responsible for extract various types of request IDs from request headers
+// and plumbing those through the context
+type RequestID struct{}
+
+func (m *RequestID) Middleware(next http.Handler) http.Handler {
+	return &recoveryHandler{
```

Comment:
I think this is supposed to be the requestIDExtractor handler

Reply:
ooof what a blunder, let me fix.